source/_posts/2016-11-08-DroidCon-London-2016.md

```diff
@@ -0,0 +1,98 @@
+# DroidCon London 2016
+
+DroidCon London is located super close to our London office, so our Android developer Mario attended as a volunteer this year! Awesome!
+
+
+## Johnny, _Mobile Lead_
+
+
+### Day 1
+
+__What's NNNNNNNNew in Android Security?__ _by Scott Alexander-Bown_
+
+Turned out to be a great talk! Very obvious that Scott have spent some time with the nitpicks and dark corners of implementing security and encryption on Android. Followed him on twitter after this talk! (@scottyab)
+
+Also the first time I heard about the SafetyNet API. A Google API that checks for a device's security state and whether it's rooted or not. Scott created a helper around that:
+
+[https://github.com/scottyab/safetynethelper](https://github.com/scottyab/safetynethelper)
+
+Further info:
+[https://developer.android.com/training/safetynet/index.html]
+(https://developer.android.com/training/safetynet/index.html)
```

Comment:
- This should look nicer, eg. "further info [here]() and [here]()." instead of the links